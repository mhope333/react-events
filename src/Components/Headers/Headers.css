.ui.block.header {
  background-color: rgb(100, 52, 128);
  color: white;
  padding-bottom: 20px;
}
.ui.huge.header {
  font-family: 'billabongregular';
  text-align: center;
  font-weight: 100;
  font-size: 10rem;
  letter-spacing: -1px;
  color: white;
  text-decoration: none;
  padding-top: 15px;
}
@font-face {
  font-family: 'billabongregular';
  src: url("https://cdn.rawgit.com/milktronics/beaglegr.am/master/public/fonts/billabong-webfont.eot");
  src: url("https://cdn.rawgit.com/milktronics/beaglegr.am/master/public/fonts/billabong-webfont.eot?#iefix") format('embedded-opentype'), url("https://cdn.rawgit.com/milktronics/beaglegr.am/master/public/fonts/billabong-webfont.woff") format('woff'), url("https://cdn.rawgit.com/milktronics/beaglegr.am/master/public/fonts/billabong-webfont.ttf") format('truetype'), url("https://cdn.rawgit.com/milktronics/beaglegr.am/master/public/fonts/billabong-webfont.svg#billabongregular") format('svg');
  font-weight: normal;
  font-style: normal;
}