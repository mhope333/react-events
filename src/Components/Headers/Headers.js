import React, {Component} from 'react';
import {Header, Icon} from 'semantic-ui-react';
import './Headers.css';

class Headers extends Component {
  render() {
  return <div>
    <Header as='h3' block>
      <Icon name='lightning'/>MH Events
    </Header>
  </div>
  }
}

export default Headers;