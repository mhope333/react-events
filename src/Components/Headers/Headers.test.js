import React from 'react';
import ReactDOM from 'react-dom';
import {cleanup} from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import Headers from './Headers';

afterEach(() => {
  cleanup();
});

describe('Headers component', () => {
  it('should render okay', () => {
    const div = document.createElement("div");
    ReactDOM.render(<Headers></Headers>, div);
  });
  it('html should match snapshot', () => {
    const tree = renderer.create(<Headers/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

