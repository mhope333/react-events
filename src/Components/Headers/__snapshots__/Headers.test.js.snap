// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headers component html should match snapshot 1`] = `
<div>
  <h3
    className="ui block header"
  >
    <i
      aria-hidden="true"
      className="lightning icon"
      onClick={[Function]}
    />
    MH Events
  </h3>
</div>
`;
